editor/pot_generator.cpp

```diff
@@ -99,25 +99,27 @@ void POTGenerator::_write_to_pot(const String &p_file) {
 		return;
 	}
 
-	String project_name = GLOBAL_GET("application/config/name");
+	String project_name = GLOBAL_GET("application/config/name").operator String().replace("\n", "\\n");
 	Vector<String> files = GLOBAL_GET("internationalization/locale/translations_pot_files");
 	String extracted_files = "";
 	for (int i = 0; i < files.size(); i++) {
-		extracted_files += "# " + files[i] + "\n";
+		extracted_files += "# " + files[i].replace("\n", "\\n") + "\n";
 	}
 	const String header =
-			"# LANGUAGE translation for " + project_name + " for the following files:\n" + extracted_files +
+			"# LANGUAGE translation for " + project_name + " for the following files:\n" +
+			extracted_files +
 			"#\n"
-			"# FIRST AUTHOR < EMAIL @ADDRESS>, YEAR.\n"
+			"# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.\n"
 			"#\n"
 			"#, fuzzy\n"
 			"msgid \"\"\n"
 			"msgstr \"\"\n"
 			"\"Project-Id-Version: " +
-			project_name + "\\n\"\n"
-						   "\"MIME-Version: 1.0\\n\"\n"
-						   "\"Content-Type: text/plain; charset=UTF-8\\n\"\n"
-						   "\"Content-Transfer-Encoding: 8-bit\\n\"\n";
+			project_name +
+			"\\n\"\n"
```

Comment:
Style-wise this could probably be on the same line.

Reply:
Do you mean like this:

```c++
			"\"Project-Id-Version: " + project_name +
			"\\n\"\n"
			"\"MIME-Version: 1.0\\n\"\n"
			"\"Content-Type: text/plain; charset=UTF-8\\n\"\n"
			"\"Content-Transfer-Encoding: 8-bit\\n\"\n";
```

Or should I revert this part?

Reply:
`clang-format` requires string literal concatenation (without the `+` operator) to continue in the same column as previous literal.

Reply:
Ah, that's why it was like that before. Damn, that sucks. I guess the first line works without triggering clang-format because the next line is a variable, not a literal?